### PR TITLE
Added theming changes for the file upload fields.

### PR DESCRIPTION
**Pull request by meirege**
## Screenshots (if appropriate):
Screenshot 2019-11-07 at 14 47 26
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: translations/nl.po [hunk 1]
```diff
@@ -298,7 +298,7 @@ msgstr "Optioneel"
 
 #: themes/contrib/gent_base/templates/core/form/input--file.html.twig:18
 msgid "Select your files here to upload"
-msgstr "Selecteer bestanden om te uploaden"
+msgstr "Selecteer bestanden om te opladen"
```
Comment (hstandaert) [inline translations/nl.po]: This doesn't really sound like decent Dutch to me 🤔

Comment (hstandaert) [reply translations/nl.po]: Much better

Comment (meirege) [reply translations/nl.po]: Fixed 🤓

**Review by hstandaert — APPROVED** [review]
Overall clean implementation 👍

**Pull request merged**